torchtext/experimental/datasets/raw/common.py

```diff
@@ -1,6 +1,14 @@
 import torch
 
 
+def check_default_set(data_select, target_select=('train', 'test', 'valid')):
+    if isinstance(data_select, str):
+        data_select = (data_select)
+    if not set(data_select).issubset(set(target_select)):
+        raise TypeError('Given data selection {} is not supported!'.format(data_select))
```

Comment:
You can add which ones are actually supported.

Reply:
Done